### ISSUE TITLE
Error with gulp task 

### ISSUE THREAD

**Issue by svzosorio** [google/material-design-lite#4291]
Actually when I try to execute a default task in gulp for compile my fork with my custom colors y have an error with task lint:sourses.

![screen shot 2016-04-21 at 12 53 50 pm](https://cloud.githubusercontent.com/assets/5873627/14719125/1e08f5d4-07c0-11e6-89d3-50b581d5f7db.png)

**Comment by judearasu** [comment]
me to find the same error, when i try to install the master version

**Comment by svzosorio** [comment]
@judearasu what is the stable branch?

**Comment by eriktrom** [comment]
yeah, eslint is borked a bit - take out `['lint:sources', 'lint:aux']` from this line https://github.com/google/material-design-lite/blob/05db1a21d75393a096fdbf3ccc14f78556364525/gulpfile.babel.js#L131 and it'll build, serve, etc

**Comment by jiaming10** [comment]
@svzosorio, I think the lint error is quite easy to fix. By using @eriktrom's method, MDL will build files succesfully, but you can't serve the site correctly through `gulp serve`.

Below is the steps to fix the lint errors, note that you actually have 3 errors.
For the first two error, please open `material-design-lite\src\mdlComponentHandler.js`, and navigate to line 233, remove the single quotes for that line. Example: ![Error](https://cloud.githubusercontent.com/assets/8718138/15187537/0987646e-17d4-11e6-8af6-d57e12ab91a5.png)
Into:
![Fixed](https://cloud.githubusercontent.com/assets/8718138/15187654/6fe36050-17d4-11e6-94a2-27c6c1655450.png)
Do the same for line 367.

Then, open `material-design-lite\src\layout\layout.js`, and go to line 569, and delete that blank line. Just like this:
![Error](https://cloud.githubusercontent.com/assets/8718138/15187763/06ea33ac-17d5-11e6-832b-1bfe73299774.png)
Into this: 
![Fixed](https://cloud.githubusercontent.com/assets/8718138/15187769/0ee242c0-17d5-11e6-8892-81f6d910508f.png)

Your gulp task should be able to complete without error now.

**Comment by svzosorio** [comment]
Thank you very much, I'll be trying again tomorrow and tell them how I was

**Comment by jiaming10** [comment]
@svzosorio, or you can try download the latest master branch, because referring to pull request #4351, the fix has been merged into.
This is the [merge commit](https://github.com/google/material-design-lite/commit/828cb52e1c8ccf41e0baac1075e9b398bb0430b0) for another reference .

**Comment by svzosorio** [comment]
Thanks @jiaming10
